Model/Resizer.php

```diff
@@ -150,7 +161,42 @@ public function resizeAndGetUrl(string $imageUrl, $width, $height, array $resize
             $this->logger->addError("Staempfli_ImageResizer: could not resize image: \n" . $e->getMessage());
         }
 
-        return $resultUrl;
+        return $resultUrl ?: $default;
+    }
+
+    /**
+     * Return relative path of resized image
+     *
+     * @return string
+     */
+    public function getRelativePath()
+    {
+        return $this->getRelativePathResizedImage();
```

Comment:
Just make the original method public. IMHO it does not make much sense to wrap it inside another method, just to make it public

Reply:
Sure, I just thought this was a better public method name for it. This way the internal one could get deprecated in time.